cli/Application/Command/Get/Project/Issues.php

```diff
@@ -47,6 +47,15 @@ class Issues extends Project
 	protected $issues = array();
 
 	/**
+	 * Status of issues.
+	 *
+	 * @var array
+	 *
+	 * @since  1.0
+	 */
+	protected $issuesStatus = array();
```

Comment:
That look and feels very good. I thought to use `[]` before but later think that it might won't work with older php version so type `array()`. :smile: